labs-ios-starter/Surveys/View Controllers/MemberListViewController.swift

```diff
@@ -50,23 +44,27 @@ class ProfileListViewController: UIViewController {
             }
             
             profileDetailVC.isUsersProfile = false
-            profileDetailVC.profile = profileController.profiles[indexPath.row]
+            profileDetailVC.profile = topic?.members?.allObjects[indexPath.row] as? Member
```

Comment:
I'm not sure this should be an NSSet, unless it's for CoreData purposes?

Reply:
We only have a CoreData model to work with and I haven't taken the time to add convenience methods to make the NSSets behave like arrays. If we have time to, that'd be great, but CoreData only gives us an NSSet to work with for one->many relationships